.hlint.yaml

```
@@ -70,6 +70,7 @@
 - ignore: { name: Use lambda case }
 - ignore: { name: Use bimap }
 - ignore: { name: Use uncurry }
+- ignore: { name: Redundant id }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)